doc/changelog/10-standard-library/10445-constructive-reals.rst

```diff
@@ -0,0 +1,7 @@
+- Define constructive real numbers by Cauchy sequences of rational numbers.
+  Redefine classical real numbers as a quotient of constructive real numbers.
+  Move proofs of constructive theorems of RIneq.v into the new RIneq_constr.v file,
+  port them back into RIneq.v by the quotient functions.
```

Comment:
You can advertise your work better than this! Here it looks like a commit message.
I would be more specific, and use a different tense. Something like:

>New module `Reals.ConstructiveCauchyReals` defines constructive real numbers by Cauchy sequences of rational numbers. Classical real numbers are now defined as a quotient of these constructive real numbers, which significantly reduces the number of axioms needed (see `Reals.Rdefinitions` and `Reals.Raxioms`), while preserving backward compatibility.

I would not bother with the last sentence from your current changelog entry: this really looks more like a technical detail.

Reply:
Done